### PR TITLE
New(tests): Basic tests (login, logout) for RHOSR

### PR DESCRIPTION
**Pull request by rkubis**

### CODE REVIEW

**Review by kornys — CHANGES_REQUESTED** [review]
please remove duplicate test

File: tests/rhosr/basic.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  await page
+    .getByRole('region', { name: 'Service Registry' })
+    .getByRole('link', { name: 'Service Registry Instances' })
+    .click();
+
+  await expect(page).toHaveTitle(/Service Registry | Red Hat OpenShift Application Services/);
+  await expect(page.getByRole('heading', { name: /Service Registry Instances/ })).toBeVisible();
+});
+
+test('perform login and logout', async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (rkubis) [reply tests/rhosr/basic.spec.ts:24]: Test in parent directory is a little bit RHOSAK specific, it tests that user is able to see landing page of RHOSAK.

This duplicate test is RHOSR specific, it tests that user is able to see landing page of RHOSR.

Probably two options here:

1. Rewrite both tests to be product-unspecific.
2. Rename test(s) to something like "landing page present/shown".

[the remark this answers is not in the document]
Comment (kornys) [reply tests/rhosr/basic.spec.ts:24]: Well the first test do a login which is ok
Im talking about the second one which is definitely the same.

[the remark this answers is not in the document]
Comment (rkubis) [reply tests/rhosr/basic.spec.ts:24]: Ah, you are right, I am going to remove this duplication. Sorry, I thought that comment is about first one/both tests.

**Review by kornys — APPROVED** [review]

**Review by Frawless — APPROVED** [review]

**Pull request merged**